src/tactic/ui/tools/custom_layout_edit_wdg.py

```diff
@@ -891,7 +892,7 @@ def get_display(my):
             text.add_style("font-family: courier")
             html = ''.join(htmls)
             html = html.replace( "&amp;", "&")
-
+            html = re.sub(r'(\<\w+\>)\s(\</\w+\>)',r'\1\2', html, re.MULTILINE)
```

Comment:
Ok done.

The limitation of this new formatting is that a user cannot put a single space between two tags.
If there is more than one space, there is no auto formatting. 

The Test button calls CustomLayoutEditSaveCmd.build_xml to get code. This means that it was use the same code as if the user pressed save.